test/conformance/tests/sriov_operator.go

```diff
@@ -216,7 +220,7 @@ var _ = Describe("[sriov] operator", func() {
 				Eventually(func() error {
 					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
 					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: "test-apivolnetwork", Namespace: namespaces.Test}, netAttDef)
-				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+				}, time.Duration(10+snoTimeout*110)*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
```

Comment:
Why you need the cast here?
It was working with 10.
I guess if you get compilation errors, you can declare snoTimeout as time.Duration

Reply:
Done